### PR TITLE
Simran ssk/aws regional support

### PR DESCRIPTION
**Pull request by simran-ssk**
- Added support so the tool does not ingest all aws accounts in Tenable.io. It will now only ingest the primary account defined in `aws_account_id` argument and any additional accounts defined in the comma delimited list argument `aws_accounts`.

- Bugfix resource the parsing of resource ARNs. The resource ARN `aws_owner_id` needs to be derived from the asset meta data.

- Changed sechubingestor to only ingest regional assets. Security Hub is a regional service.

- Updated README with instructions for --aws-account argument.

### CODE REVIEW

File: sechubingest.py [hunk 1]
```diff
@@ -213,15 +222,15 @@ def _transform_finding(self, vuln):
                 'Product': vuln.get('plugin').get('cvss_base_score',
                                 sevmap[vuln.get('severity_default_id', 0)]),
                 'Normalized': int(vuln.get('plugin').get('cvss_base_score',
-                                sevmap[vuln.get('severity_default_id', 0)]) * 4),
```
Comment (SteveMcGrath) [inline sechubingest.py]: Setting the top possible vuln rating to 40 for the normalized score is based on feedback from Amazon.  It was originally as you had recommended

From Amazon:
> Findings that are associated with issues that could result in future compromises (e.g., vulnerabilities, configuration weaknesses, exposed passwords). This generally aligns to the Software and Configuration Checks namespace under a finding's type. Suggested score: 1–39. These findings should receive a Low severity label.

**Comment by SteveMcGrath** [comment]
Generally speaking, we hear the opposite about account IDs.  most folks want to just ingest everything, not cherrypick specific account ids.

As for regionality wouldn't it make more sense then to then create boto3 clients on the fly for the regions as needed and ingest to the appropriate region?

File: sechubingest.py (hunk 1 above)
Comment (simran-ssk) [reply sechubingest.py]: I've updated this per your feedback. I updated the tests as well.

**Comment by simran-ssk** [comment]
I changed the default behavior to ingest findings from all accounts unless otherwise specified using the `-aws-accounts` argument. 

As for the regionality I can cut a separate PR for that. I was thinking to make the transforms list into a defaultdict(list) and then appending findings per region or something similar.

**Comment by simran-ssk** [comment]
Are you ok accepting the changes I’ve purposed?

**Comment by SteveMcGrath** [comment]
You've actually highlighted some fundamental changes that need to be made overall.  I've been in contact with AWS on writing a v2.

**Pull request closed without merging**